h/browser/chrome/karma.config.js

```diff
@@ -32,6 +41,9 @@ module.exports = function(config) {
       '../../static/scripts/blocklist.js'
     ],
 
+    proxies: {
+      "/settings.json": "http://localhost:9877/base/test/settings.json"
```

Comment:
You don't need to specify the hostname. `/base/test/settings.json` appears to work, and that relieves you from the need to change the port, too.